.github/workflows/build-and-deploy.yml

```diff
@@ -39,14 +39,14 @@ jobs:
           name: website-build
           path: /tmp/website-build
       - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
       - name: Deploy website
         if: github.event_name == 'push'
         run: |
           git config --global user.name SymbiFlow
           git config --global user.mail robot@mith.ro
           git config --global push.default simple
-          git fetch origin --tags
```

Comment:
We want `--tags` so that `git describe` gives the right output.

Reply:
Option `fetch-depth: 0` was implemented for this use case. See:

https://github.com/actions/checkout/#usage
```
    # Number of commits to fetch. 0 indicates all history for all branches and tags.
    # Default: 1
    fetch-depth: ''
```

and https://github.com/actions/checkout/issues/250#issuecomment-634799150.